src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -137,20 +137,22 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RuntimeFrameworkVersion Condition="'$(TargetLatestRuntimePatch)' != 'true' ">$(DefaultNetCorePatchVersion)</RuntimeFrameworkVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(DisableImplicitFrameworkReferences)' != 'true'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(DisableImplicitFrameworkReferences)' != 'true'">
     <!-- This property is different than MicrosoftNETPlatformLibrary.  MicrosoftNETPlatformLibrary is
          used to trim the dependencies published with a framework-dependent app, and is overridden
          by ASP.NET packages.
-         
-         The ImplicitPlatformPackageIdentifier is the package that is actually implicitly
+
+         The ImplicitPlatformPackages is the list of packages that are actually implicitly
          referenced by the SDK, and is passed into the ResolvePackageAssets task
-         to verify that the restored version of the package matches the
-         RuntimeFrameworkVersion -->
-    <ImplicitPlatformPackageIdentifier>Microsoft.NETCore.App</ImplicitPlatformPackageIdentifier>
+         to verify that the restored versions of the packages matches the expected versions
+         -->
+    <ImplicitPlatformPackages Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)" />
+  </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(DisableImplicitFrameworkReferences)' != 'true'">
     <VerifyMatchingImplicitPackageVersion Condition="'$(VerifyMatchingImplicitPackageVersion)' == ''">true</VerifyMatchingImplicitPackageVersion>
   </PropertyGroup>
-  
+
```

Comment:
There are a bunch of whitespace changes in this file, please revert them.

Reply:
My editor is set up to remove trailing white spaces. Why do we want to retain them?

Reply:
You can ignore it with ?w=1

Reply:
At least put whitespace-only changes in their own commits. It breaks git blame to touch lines for no reason.

Reply:
Yes, git blame has -w, but various things invoke it for you. It's really better not to mix in style changes unrelated to a PR. Thanks.

Reply:
It's a bit hard to separate it out after the fact. I will keep that in mind in the future. I will revert the changes to the other file with only whitespace changes.

Reply:
Oh actually there might be a way to do it. Let me try.